zerver/decorator.py

```diff
@@ -3,8 +3,7 @@
 from collections.abc import Callable, Sequence
```

Comment:
This first commit should not have a "Fixes" tag, since merging just this commit will not fix the issue.